package.json

```diff
@@ -69,6 +69,7 @@
     "lint": "eslint lib test",
     "format": "prettier --write eslint.config.mjs \"{lib,test}/**/*.js\"",
     "docs": "remark README.md 'docs/**/*.md' '.github/*.md' -o",
+    "test2": "NODE_DEBUG=release-it* ava --no-worker-threads && installed-check --ignore ava --ignore nock",
```

Comment:
Please either remove this or give the script a more descriptive name

Reply:
@webpro that was an oversight on my behalf. will fix!